### PR TITLE
Generates basic but readable bar chart of the benchmark results.

### PR DESCRIPTION
**Pull request by SekouDiaoNlp**

### CODE REVIEW

**Comment by juancarlospaco** [comment]
- ReSync with the main branch, so I can merge.
- Add the PNG to the README.md so people can see it.
- Add yourself to the authors list somehow.

**Comment by SekouDiaoNlp** [comment]
> * ReSync with the main branch, so I can merge.
> * Add the PNG to the README.md so people can see it.
> * Add yourself to the authors list somehow.
> * No need to copy all the files inside `dist/` thats done automatically by https://github.com/juancarlospaco/faster-than-csv/blob/master/package4pypi.sh#L5

Will do.

File: dist/faster_than_csv/PKG-INFO [hunk 1]
```diff
@@ -43,4 +43,4 @@ Classifier: Programming Language :: Python :: Implementation :: CPython
 Classifier: Topic :: Software Development
 Requires-Python: >=3.8
 Description-Content-Type: text/markdown
-Version: 21.03.03
+Version: 21.03.04
```
Comment (juancarlospaco) [inline dist/faster_than_csv/PKG-INFO:46]: This version is the current date `YY.MM.DD`, kinda like the version that PIP uses.

Comment (SekouDiaoNlp) [reply dist/faster_than_csv/PKG-INFO:46]: OK

**Review by SekouDiaoNlp — COMMENTED** [review]
seems ok

**Review by SekouDiaoNlp — COMMENTED** [review]
Seems ok.

**Comment by SekouDiaoNlp** [comment]
> * ReSync with the main branch, so I can merge.
Done.
> * Add the PNG to the README.md so people can see it.
Done.
> * Add yourself to the authors list somehow.
Done. Added a Contributors section in the README.
> * No need to copy all the files inside `dist/` thats done automatically by https://github.com/juancarlospaco/faster-than-csv/blob/master/package4pypi.sh#L5
OK

**Review by juancarlospaco — APPROVED** [review]

**Pull request merged**